### DISCUSSION TITLE
Is there any plan for bdd2voc conversion.

### DISCUSSION THREAD

**Discussion by sainisanjay** [bdd100k/bdd100k#41] [Q&A]
Since, in the repo already bdd to coco conversion script is available, it would be great, if bdd2voc conversion script will be added to repo.
Although, many tools and GitHub repository is available in the Google but None of supports the new BDD 2020 Detection labels.

**Comment by fyu** [comment]
We are considering the option, but a lot of tools are already available to convert coco to voc. Google will tell you some of them.

**Reply by sainisanjay** [reply]
Could you please help me any of the tool which supports your coco conversion file into voc.

**Reply by XiaLiPKU** [reply]
Maybe this can help you currently: https://github.com/alicranck/coco2voc

**Reply by sainisanjay** [reply]
Thanks @XiaLiPKU for link but unfortunately it didn't work with this repo coco conversion to voc. I am trying to convert "`det_v2_train_release_coco.json`"(this file contain all annotations in coco format which i got by running `to_coco.py`) into individual filename Pascal voc `.xml` files.
